README.md

```diff
@@ -1,90 +1,274 @@
+  </ul>
+</details>
 
-## Built With
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
 
-<ul>
-  <li><a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby 3.1</a></li>
-</ul>
+<!-- Features -->
 
+### Key Features <a name="key-features"></a>
 
-## Getting Started 
+> Describe between 1-3 key features of the application.
 
-> To get a local copy up and running follow these simple steps.
+- **[Add new students or teachers]**
+- **[Add new books]**
+- **[Save records of who borrowed a given book and when.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
-- Ruby
-- IDE
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install 
+```
+ -->
 
 ### Setup
 
-1. Clone this project on your local machine using the following command on your Git Bash: `git clone https://github.com/arabiu033/school_library.git`
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/arabiu033/school_library.git
+  cd school_library
+  bundle install
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  run bundle install
+```
 
-2. Browse to project directory `cd school_library`
+### Usage
 
-3. Run `bundle install` to install the dependencies
+To run the project, execute the following command:
 
+<!--
+Example command:
 
-### Available Scripts
+```sh
+  irb
+  require_relative(file path)
+```
+--->
 
-In the project directory, you can run:
+### Run tests
 
-- `IN PROGRESS`
+To run tests, run the following command:
 
-  Runs the app on your bash
+<!--
+Example command:
 
-- `rubocop`
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
 
-  Launches the test runner.
+### Deployment
 
-## Authors
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
 
 👤 **Aminu Rabiu**
 
 - GitHub: [@arabiu033](https://github.com/arabiu033)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/larabiu033)
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
 
-## Contributing 
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```

Comment:
- [ ] Kindly state three specific features you would be adding to the project.

Reply:
Sorry about that :heart: Implemented now :+1: